mautic/mautic
# Company segments

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | n
| New feature? | y
| Related user documentation PR URL | pending
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR adds the ability to filter contacts based on company fields. You can now assign a contact to a company or companies, and based on filters in a segment all contacts belonging to companies that meet the filtering criteria together with contact filtering criteria will be added to  a segment. So for example if a contact lives in Brazil and is belongs to company B then that contact will be added to the segment. 

#### Steps to test this PR:
1. Apply PRs #2403 and #2424 together with this PR
2. create some companies, add contacts to a company, then create a segment with filters that would apply to your values.
3. you should now have contacts in your segment that meet both contact filters and company filters criteria.
